packages/meld.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Meld < Package
+  description 'Meld is a visual diff and merge tool targeted at developers.'
+  homepage 'https://meldmerge.org/'
+  version '3.22.0'
+  license 'GPL-2'
+  compatibility 'all'
```

Comment:
`compatibiliy 'x86_64'` or do we know if this is compatible with all architectures?